leones/src/main/java/globantWorkshop/services/implementation/BookService.java

```diff
@@ -30,15 +30,19 @@
 
     @Override
     public ArrayList<Book> getAllBooks() {
-        //Should Be implemented
         List<Book> books = new ArrayList<Book>();
+        try {
+            books = bookDao.getAllBooks();
+        } catch (Exception ex) {
```

Comment:
You should catch more specific exceptions than just Exception and there's no point in catching and throwing the same exception without doing something with it, for instance log the error.